docs/pages/includes/compatibility.mdx

```diff
@@ -14,5 +11,6 @@ apply:
   You must upgrade to 7.x.x first.
 - Proxy Service instances and agents do not support Auth Service instances that
   are on an older major version, and will fail to connect to older Auth Service
-  instances by default. You can override version checks by passing
+  instances by default. For example, an 8.x.x Proxy Service or agent is not
+  compatible with an 7.x.x Auth Service. You can override version checks by passing
   `--skip-version-check` when starting agents and Proxy Service instances.
```

Comment:
Can we also stress a bit more that `--skip-version-check`, especially on the proxy, is very much not guaranteed to work?